aggregator/src/datastore.rs

```diff
@@ -838,6 +838,30 @@ impl<C: Clock> Transaction<'_, C> {
         )?)
     }
 
+    /// Retrieves the HPKE config IDs currently in use for the specified task.
+    pub async fn get_hpke_config_ids_for_task(
```

Comment:
Could we use `get_task`, then read the HPKE config IDs out of the full structure? That would save a little bit of extra code here, at a probably-negligible cost to performance.

Reply:
I deleted this because we don't need to query existing HPKE config IDs per your observation in `janus_cli.rs`, but I'll remember your remark about trading off maintenance burden vs. hypothetical performance win.